Converters/Option1d.py

```diff
@@ -281,13 +287,13 @@ def convertShapeFile(shpFilename, cdbInputDir, cdbOutputDir):
 
     #Read all the feature records from the DBF at once (using GDAL)
     #copyFeaturesFromShapeToGeoPackage(shpFilename,outputGeoPackageFile)
-    fClassRecords = converter.readDBF(fcAttrName)
+    #fClassRecords = converter.readDBF(fcAttrName)
     #Read Featureclass records
     featureTableName = converter.getFeatureAttrTableName(shpFilename)
     copyFeaturesFromShapeToGeoPackage(shpFilename,outputGeoPackageFile)
     #convertSHP(sqliteCon,shpFilename,outputGeoPackageFile, fClassRecords, True)
     sqliteCon = sqlite3.connect(outputGeoPackageFile)
-    if(createExtendedAttributesTable(sqliteCon,shpFilename)):
+    if(0 and createExtendedAttributesTable(sqliteCon,shpFilename)):
```

Comment:
Ryan, could you please explain why you want to disable the above `if` statement ?

Reply:
This code crashes on Yemen (Tiles\N12\E044\201_RoadNetwork\L00\U0\N12E044_D201_S002_T003_L00_U0_R0.shp).  It is trying to get the CNAM attribute, but from an extended attribute DBF file where one doesn't exist.  Since it wasn't converting correctly anyway, I disabled it.